app/lib/features/onboarding/pages/start_page.dart

```diff
@@ -16,229 +17,233 @@ class StartPage extends StatefulWidget {
 
 class _StartPageState extends State<StartPage> {
   // Variables
-  PageController _pageController = PageController(initialPage: 0);
+  final PageController _pageController = PageController(initialPage: 0);
 
   int _pageIndex = 0;
 
   // OnBoarding content list
-  final List onBoardingPages = [
-    OnBoarding(
-      image: 'assets/images/spaces_onboard.png',
-      title: RichText(
-        textAlign: TextAlign.center,
-        text: const TextSpan(
-          text: 'Organize & Collaborate Through ',
-          style: TextStyle(color: Colors.green, fontSize: 24),
-          children: <TextSpan>[
-            TextSpan(
-              text: 'Spaces',
-              style: TextStyle(
-                fontWeight: FontWeight.bold,
-                decoration: TextDecoration.underline,
+  List onBoardingPages = [];
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    onBoardingPages = [
```

Comment:
same here, why not just in `build`?

Reply:
Same as answer mentioned in above comment.